drivers/flash/flash_simulator.c

```diff
@@ -147,7 +149,11 @@ STATS_NAME_END(flash_sim_thresholds);
 static uint8_t *mock_flash;
 static int flash_fd = -1;
 static const char *flash_file_path;
-static const char default_flash_file_path[] = "flash.bin";
+#define DEFAULT_FLASH_FILE_PATH "flash.bin"
```

Comment:
I think this should be moved somewhere on top of the file, under includes, as a visibly defined default.
Having it here to just assign it to something in  the next line and not using it anywhere seems to be just waste of line.
This should be more clearly advertised as something that impacts internals of the source.

Reply:
done (note it is used below in the command line argument description)